Userland/Libraries/LibGUI/TextEditor.h

```diff
@@ -123,6 +123,7 @@ class TextEditor
     TextRange normalized_selection() const { return m_selection.normalized(); }
 
     void insert_at_cursor_or_replace_selection(StringView);
+    void replace_all_text_while_keeping_undo_stack(StringView text);
```

Comment:
The `while_keeping_undo_stack` suffix here is a little bit misleading, as it sounds like *nothing* happens to the undo stack.
`without_resetting_undo_stack` perhaps?